### PR TITLE
Rat Christmas Hats

### PR DESCRIPTION
**Pull request by ParadoxSpace**
I'm late, but.
Puts tiny Christmas hats on the station rats.
image

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

File: html/changelogs/paradoxspace-ratmas.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#   experiment
+#   balance
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: ChangeMe
```
Comment (fernerr) [inline html/changelogs/paradoxspace-ratmas.yml]:
```suggestion
author: ParadoxSpace
```

**Comment by KasparoVy** [comment]
Beautiful.

**Comment by Arrow768** [comment]
That´s gonna get a testmege on Christmas since its easier than creating a revert PR afterwards.

**Comment by Arrow768** [comment]
Has been testmerged.
Now its gone.

**Pull request closed without merging**